acceptance/tests/fixtures/cases/api-gateways/jwt-auth/jwt-route-filter.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: HTTPAuthFilter
```

Comment:
Should this be `HTTPRouteAuthFilter` based on #2734 ?

Reply:
yup it should be!